src/pages/Checkout.vue

```diff
@@ -0,0 +1,406 @@
+              :ticketType="ticket.ticket" :confirmationCode="`SB11054${ticket.id}`"
+              maxWidth="300px" />
+          </div>
+        </transition-group>
+      </template>
+
+      <div class="tickets-container empty-state-container" :class="{ 'col-12': mobileView, 'col-8': !mobileView }" v-if="tickets.length == 0">
+        <div class="empty-state-text">
+          <p>You haven't selected any tickets to purchase yet. <a @click="addTicket()">Add a ticket now.</a></p>
+        </div>
+      </div>
+
+      <div class="sidebar-container ticket-selection" :class="{ 'col-12': mobileView, 'col-4': !mobileView }" v-if="!isCurrentlyEditing">
+        <div class="ticket-item" v-for="(ticket, ticketIndex) in tickets" :key="ticket.id" @click="editTicket(ticketIndex)">
+          <h2>{{ `${ticket.firstName} ${ticket.lastName}` }}</h2>
+          {{ ticket.ticket }} &middot; {{ ticket.meal }}
+        </div>
+        <button class="full-width extra-margin-top secondary" @click="addTicket()">Add Another Ticket</button>
+        <button class="full-width" @click="goToPayment()" v-if="tickets.length > 0">Continue to Payment</button>
+      </div>
+
+      <div class="col-4 sidebar-container ticket-form" :class="{ 'col-12': mobileView, 'col-4': !mobileView }" v-else>
+        <h2>Ticket Information</h2>
+        <p class="footnote show-label">Ticket Type</p>
+        <select v-model="currentTicket.ticket" class="full-width">
+          <option>General Ticket</option>
+          <option>UW Student Ticket</option>
+        </select>
+
+        <p class="footnote show-label">Meal Type</p>
+        <select v-model="currentTicket.meal" class="full-width">
+          <option disabled value="">Select a meal</option>
+          <option>No Meal</option>
+          <option>Vegan</option>
+          <option>Non-Vegan</option>
+        </select>
+
+        <h2 class="extra-margin-top">Ticket Holder</h2>
+        <p class="footnote" :class="{ 'show-label': !!currentTicket.firstName }">First Name</p>
+        <input type="text" placeholder="First Name" class="full-width" v-model="currentTicket.firstName">
+        <p class="footnote" :class="{ 'show-label': !!currentTicket.lastName }">Last Name</p>
+        <input type="text" placeholder="Last Name" class="full-width" v-model="currentTicket.lastName">
+
+        <p v-if="showError" class="error extra-margin-top">Make sure you've filled out all parts of the form before saving.</p>
+        <button class="full-width" :class="{ 'extra-margin-top': !showError }" @click="saveTicket()">Save</button>
+        <button class="full-width secondary" @click="cancelTicket()" v-if="tickets.length > 1">{{ creatingTicket ? 'Cancel' : 'Delete Ticket' }}</button>
+      </div>
+    </div>
+
+    <div class="row" v-else>
+      <p>Screen where you submit payment details will come soon, based on Jenny's payments work :)</p>
+    </div>
+  </div>
+</template>
+
+<script>
+const DEMO_MODE = false;
+const MOBILE_MAX_WIDTH = 1350;
```

Comment:
Could we make this constant a bit smaller, maybe ~500 or 600? Or would that look too weird on devices like tablets? I noticed that on my computer, the mobile view rendered with a relatively large screen width but I think it'd be awesome to have the desktop view for as long as possible so more people can enjoy it :-D

Reply:
We decided to not change this right now. To accomplish this, tickets in the spotlight view would need to compress "under" each other, which is somewhat considerable work for marginal gain. Especially since the mobile spotlight view looks good on desktop, too :)